docs/book/src/plugins/extending/custom-markers.md

```diff
@@ -0,0 +1,349 @@
+# Creating Custom Markers for Unsupported File Extensions
```

Comment:
I think the documentation could be How to create Custom Markers.
For those who use Kubebuilder as a library, it would be possible to create their own custom markers, regardless of whether it is for an extension valid or not for Golang projects.

Reply:
Can we make it more generic?
Also, is it possible to do it as low verbose as possible?